arborist/server_test.go

```diff
@@ -3243,7 +3250,19 @@ func TestServer(t *testing.T) {
 				w := httptest.NewRecorder()
 				req := newRequest("POST", "/auth/mapping", nil)
 				handler.ServeHTTP(w, req)
-				assert.Equal(t, w.Code, http.StatusBadRequest, "expected a 400 response")
+				assert.Equal(t, w.Code, http.StatusOK, "expected a 200 OK")
```

Comment:
from what I can see 3117 - 2129 and 3253 - 3265 seem to be the same set of code. Would it be possible to encapsulate this behavior in a separate function?

Reply:
updated